src/server/saved_objects/client/lib/search_dsl/sorting_params.js

```diff
@@ -7,21 +7,35 @@ export function getSortingParams(mappings, type, sortField, sortOrder) {
     return {};
   }
 
-  const key = type ? `${type}.${sortField}` : sortField;
+  if (Array.isArray(type)) {
+    const rootField = getProperty(mappings, sortField);
+    if (!rootField) {
+      throw Boom.badRequest(`Unable to sort multiple types by field ${sortField}, not a root field`);
```

Comment:
The error message.. Will people know what a "root field" is? If it's not something we say anywhere else, i'd use language that is consistent...

Reply:
Up until this point, we haven't had to leak the way that these objects are mapped to the internal indices to consumers of the SavedObjectsClient yet, so I'm not aware of any existing language besides what we refer to them within the mappings "module" which is as "root properties", perhaps "root properties" would be more clear?

Reply:
Yeah, I do think `root properties` is clearer. You think of fields and properties as the same thing. I can see that.. but I sort of feel like fields are just anything under the "properties" key that is not an object datatype. But even this usage of "field" isn't totally fool-proof because there's also multi-fields fields, which actually use the key `field` (https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html), so we're really at a loss for a word to describe a non-multi-field, non object datatype thing that lives in `properties`.

I'm really not sure if `root field` or `root property` is better here.